### DISCUSSION TITLE
Quiz point

### DISCUSSION THREAD

**Discussion by akaKarthick** [czbar/ChessForge#632] [Q&A] [answered]
- How are quiz points awarded?

- What can quiz points be used for?

- Are there any plans to expand the quiz point feature in the future?

**Comment by czbar** [comment, marked as answer]
Oh, I remember writing a fairly long answer to this, but it does not seem to have been saved (forgot to push "Comment" ??).  So, here's the short version:
- quiz is an experimental feature and it is open to further design/evolution, if users find it useful in this shape or another
- points are awarded if the moves in the main line of the submitted solution match the points specified by the author of the exercise (there is some complication when the points are specified for non-main lines but that's not too relevant right now).
